openedx/core/djangoapps/discussions/views.py

```diff
@@ -156,7 +156,7 @@ def get(self, request, course_key_string: str, **_kwargs) -> Response:
         Handle HTTP/GET requests
         """
         # Return all providers always, if the user is staff
-        data = self.get_provider_data(course_key_string, show_all=request.user.is_staff)
+        data = self.get_provider_data(course_key_string, show_all=False)
```

Comment:
why remove this?

Reply:
It was updated because we wanted the same behavior for all user types.